<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChatForm</class>
 <widget class="QWidget" name="ChatForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>972</width>
    <height>599</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
	font: 9pt &quot;微软雅黑&quot;;
}

#w_Main{
	border:1px solid rgb(13,97,137);
	border-radius:2px;
	border-image: url(:/sys/back.png) 5;
	border-color: rgb(255, 255, 255);
}

#w_function{
	background-color: rgb(255, 255, 255);
	border-left:1px solid rgb(13,97,137);
}

#w_text{
	background-color: rgb(255, 255, 255);
	border:1px solid rgb(13,97,137);
}

#w_show{
	border-right:1px solid rgb(13,97,137);
}

#textBrowser_show{
	border-left:1px solid rgb(13,97,137);
}

#w_top{
	background-color: rgb(22, 154, 218);
}


#w_top QPushButton{
	border:none;
	border-radius:2px;
}

#w_top QPushButton:hover{
	border:1px solid rgb(13,97,137);
}

#w_top QPushButton#pb_minimize{
	background-image: url(:/sys/mainform_minimize.png);
}

#w_top QPushButton#pb_shutdown{
	background-image: url(:/sys/mainform_shutdown.png);
}

#w_top QPushButton#pb_nickName{
	color:#fff;
	font: 11pt &quot;微软雅黑&quot;;
}

#w_qqshow{
	border-image: url(:/sys/Chat/chat_show.jpg);
	border-right:1px solid rgb(13,97,137);
}

#w_text QPushButton:hover{
	border:1px solid rgb(13,97,137);
}

QTextEdit{
	border:0px;
}

#w_function QPushButton#pb_cut{
	background-image: url(:/sys/Chat/chat_cut.png);
	background-repeat: no-repeat;
}

#w_function QPushButton#pb_expression{
	background-image: url(:/sys/Chat/chat_expression.png);
	background-repeat: no-repeat;
}

#w_function QPushButton#pb_more{
	background-image: url(:/sys/Chat/chat_more.png);
	background-repeat: no-repeat;
}

#w_function QPushButton#pb_sendfile{
	background-image: url(:/sys/Chat/chat_sendfile.png);
	background-repeat: no-repeat;
}

#w_function QPushButton#pb_sendpic{
	background-image: url(:/sys/Chat/chat_sendpic.png);
	background-repeat: no-repeat;
}

#w_function QPushButton#pb_history{
	background-image: url(:/sys/Chat/chat_history.png);
	background-repeat: no-repeat;
}

#w_function QPushButton{
	background-color: rgb(255, 255, 255,0);
}
</string>
  </property>
  <widget class="QWidget" name="w_Main" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>981</width>
     <height>571</height>
    </rect>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>-3</x>
      <y>0</y>
      <width>971</width>
      <height>581</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>2</number>
     </property>
     <property name="bottomMargin">
      <number>20</number>
     </property>
     <item>
      <widget class="QWidget" name="w_top" native="true">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>41</height>
        </size>
       </property>
       <widget class="QPushButton" name="pb_minimize">
        <property name="geometry">
         <rect>
          <x>910</x>
          <y>10</y>
          <width>20</width>
          <height>20</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QPushButton" name="pb_shutdown">
        <property name="geometry">
         <rect>
          <x>940</x>
          <y>10</y>
          <width>20</width>
          <height>20</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QPushButton" name="pb_nickName">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>10</y>
          <width>171</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>飞翔的企鹅</string>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>2</number>
         </property>
         <property name="rightMargin">
          <number>1</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QWidget" name="w_record" native="true">
           <property name="minimumSize">
            <size>
             <width>681</width>
             <height>361</height>
            </size>
           </property>
           <widget class="QTextBrowser" name="textBrowser_show">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>681</width>
              <height>361</height>
             </rect>
            </property>
           </widget>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="w_text" native="true">
           <property name="minimumSize">
            <size>
             <width>681</width>
             <height>148</height>
            </size>
           </property>
           <widget class="QWidget" name="w_function" native="true">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>681</width>
              <height>41</height>
             </rect>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>31</height>
             </size>
            </property>
            <widget class="QPushButton" name="pb_history">
             <property name="geometry">
              <rect>
               <x>634</x>
               <y>1</y>
               <width>40</width>
               <height>40</height>
              </rect>
             </property>
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
            <widget class="QWidget" name="layoutWidget">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>211</width>
               <height>41</height>
              </rect>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <property name="spacing">
               <number>0</number>
              </property>
              <item>
               <widget class="QPushButton" name="pb_expression">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="autoFillBackground">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string/>
                </property>
                <property name="iconSize">
                 <size>
                  <width>20</width>
                  <height>20</height>
                 </size>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pb_cut">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="autoFillBackground">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pb_sendfile">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pb_sendpic">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pb_more">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
           <widget class="QTextEdit" name="textEdit_input">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>50</y>
              <width>671</width>
              <height>45</height>
             </rect>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>671</width>
              <height>45</height>
             </size>
            </property>
           </widget>
           <widget class="QWidget" name="layoutWidget">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>110</y>
              <width>660</width>
              <height>31</height>
             </rect>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <property name="spacing">
              <number>0</number>
             </property>
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeType">
                <enum>QSizePolicy::Fixed</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>558</width>
                 <height>15</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="pb_send">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>发送(S)</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QWidget" name="w_show" native="true">
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <widget class="QWidget" name="w_qqshow" native="true">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>291</width>
            <height>511</height>
           </rect>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
